### ISSUE TITLE
Running demo.sh in src/tutorial-data/config-file makes error

### ISSUE THREAD

**Issue by aqjune** [mjambon/atdgen-doc#4]
The error is like following : 

```
+ echo 'let contents = "\'
+ sed -e 's/\([\\"]\)/\\\1/g' config.atd
+ echo '"'
+ atdgen -t config.atd
+ atdgen -j -j-defaults -j-strict-fields config.atd
+ atdgen -v config.atd
+ ocamlfind ocamlopt -o config config_t.mli config_t.ml config_j.mli config_j.ml config_v.mli config_v.ml config_atd.ml config.ml -package atdgen -linkpkg
File "config_v.ml", line 38, characters 0-3:
Error: Syntax error: ')' expected
File "config_v.ml", line 26, characters 18-19:
Error: This '(' might be unmatched
```

My ocaml version is 4.02.3 .

**Comment by mjambon** [comment]
Thanks for reporting that. It looks like a bug in atdgen's code generation that for some reason wasn't detected so far.

**Comment by mjambon** [comment]
The fix is included in atdgen 1.7.2 which has submitted to opam https://github.com/ocaml/opam-repository/pull/5343
